src/log-store/src/kafka/client_manager.rs

```diff
@@ -63,9 +60,8 @@ pub(crate) struct ClientManager {
     /// Used to initialize a new [Client].
     mutex: Mutex<()>,
     instances: RwLock<HashMap<Arc<KafkaProvider>, Client>>,
+    global_index_collector: Option<GlobalIndexCollector>,
```

Comment:
Looks like this field is not assigned anywhere?

Reply:
Used in the future PRs.